src/algorithms/math/fast_prime.js

```diff
@@ -0,0 +1,20 @@
+/**
+* Check if number is prime or not
+* @param  {Number} a number to be checked
+* @return {Number}  true or false
+*/
+const isPrime = (a) => {
+  for (let i = 2; i <= Math.sqrt(a); i+= 1) {
```

Comment:
If you check `if (a%2 === 0) return false;` before this loop then you only need to check odd numbers here. So the loop would be:
```
const root = Math.sqrt(a); // cache the root
for (let i = 3; i <= root; i += 2) {
```

Reply:
For the second condition i think we must check even numbers greater than 2
because 2 itself is prime